packages/astro/src/core/create-vite.ts

```diff
@@ -43,21 +42,7 @@ const ALWAYS_NOEXTERNAL = new Set([
 	'@nanostores/preact',
 	// fontsource packages are CSS that need to be processed
 	'@fontsource/*',
-]);
-
-function getSsrNoExternalDeps(projectRoot: URL): string[] {
-	let noExternalDeps = [];
-	for (const dep of ALWAYS_NOEXTERNAL) {
-		try {
-			resolveDependency(dep, projectRoot);
-			noExternalDeps.push(dep);
-		} catch {
-			// ignore dependency if *not* installed / present in your project
-			// prevents hard error from Vite!
-		}
-	}
-	return noExternalDeps;
-}
```

Comment:
Vite doesn't have an issue now if the deps are not installed. This enables removing `resolveDependency` and the `resolve` package.